src/Rules/Function/IncludeFunctionRule.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TwigCsFixer\Rules\Function;
+
+use TwigCsFixer\Rules\AbstractFixableRule;
+use TwigCsFixer\Token\Token;
+use Webmozart\Assert\Assert;
+
+/**
+ * Ensures that include function is used instead of function tag.
+ */
+final class IncludeFunctionRule extends AbstractFixableRule
+{
+    protected function process(int $tokenPosition, array $tokens): void
+    {
+        $token = $tokens[$tokenPosition];
+        if (!$this->isTokenMatching($token, Token::BLOCK_NAME_TYPE, 'include')) {
+            return;
+        }
+
+        $openingTag = $this->findPrevious(Token::BLOCK_START_TYPE, $tokens, $tokenPosition);
+        Assert::notFalse($openingTag, 'Opening tag cannot not be null.');
+
+        $closingTag = $this->findNext(Token::BLOCK_END_TYPE, $tokens, $tokenPosition);
+        Assert::notFalse($closingTag, 'Closing tag cannot not be null.');
+
+        $fixer = $this->addFixableError(
+            'Include function must be used instead of include tag.',
+            $token
+        );
+
+        if (null === $fixer) {
+            return;
+        }
+
+        $fixer->beginChangeSet();
+        $fixer->replaceToken($openingTag, '{{');
```

Comment:
You're changing `{%` to `{{`.
What if it was `{%~` or `{%-` ? (This will need to be tested)

Reply:
Thx i keep forgetting about those